machine/meta-xt-images-rcar-gen3/conf/machine/h3ulcb-cb-xt.conf

```diff
@@ -0,0 +1,13 @@
+#@TYPE: Machine
+#@NAME: H3ULCB ES2.0 Cetibox machine
+#@DESCRIPTION: Machine configuration for running xen domain on H3ULCB ES2.0 Cetibox
+
+SOC_FAMILY = "r8a7795"
+
+XT_CANONICAL_MACHINE_NAME = "h3ulcb"
+
+require conf/machine/include/rcar.inc
+
+UBOOT_MACHINE = "r8a7795_ulcb_defconfig"
```

Comment:
it should be double checked whether r8a7795_ulcb_defconfig is suitable for the machine in question.

Reply:
It seems the mentioned config was removed from the current version on review, so I assume the question is not relevant anymore